_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,422 @@
+
+To download and install the Reporting CLI from npm, run the following command to initiate installation:
+
+```
+npm i @opensearch-project/opensearch-reporting-cli
+```
+
+### Downloading and installing the Reporting CLI from OpenSearch.org
+
+You can download the `opensearch-reporting-cli` tool from the OpenSearch.org [Reporting CLI Tool](https://opensearch.org/downloads.html#opensearch-reporting-cli) section.
+
+Next, run the following command to install the .tar archive:
+
+```
+npm install -g opensearch-reporting-cli-1.0.0.tgz
+```
+
+## Creating and requesting a visualization report
+
+First, you need to get the URL for the visualization that you want to download as an image file or PDF.
+
+To generate a visualization report, you need to specify the Dashboards URL.
+
+Open the visualization for which you want to generate a report, and select **Share >  Permalinks > Generate link as Shapshot > Short URL > Copy link**, as shown in the following image.
+
+![Copy link]({{site.url}}{{site.baseurl}}/images/dashboards/dash-url.png)
+
+You will need to add the URL with the `-u` argument when you request the report in the CLI.
+
+### Example: Requesting a PNG file
+
+The following command requests a report in PNG format with basic authentication and sends the report to an email address using Amazon SES:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:1234 -e ses -s <email address>  -r <email address> -f png
+```
+
+### Example: Requesting a PDF file
+
+The following command requests a PDF file and specifies the recipient's email address:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s <email address> -r <email address> -f pdf
+```
+
+Upon success, the file will be sent to the specified email address. The following image shows an example PDF report.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+### Example: Requesting a CSV file
+
+The following command generates a report that contains all table content in CSV format and sends the report to an email address using Amazon SES transport:
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -f csv -a basic -c admin:Test1234 -e ses -s <email address> -r <email address>
+```
+
+Upon success, the email will be sent to the specified email address with the CSV file attached.
+
+## Scheduling reports with the cron utility
+
+You can use the cron command-line utility to initiate a report request with the reporting CLI that runs periodically at any date or time interval. Follow the cron expression syntax to specify the date and time that precedes the command that you want to initiate.
+
+To learn about the cron expression syntax, see [cron](https://en.wikipedia.org/wiki/Cron).
```

Comment:
Is wikipedia considered a primary source? If not, should we be linking to it?

Reply:
It's a Linux/Unix built-in utility. I chose wikipedia because it's not tied to a specific flavor of Linux. will ask the team. other options: Oracle Linux  - [Use crontab utility](https://docs.oracle.com/en/learn/oracle-linux-crontab/index.html). . .
Ubuntu [Cron how to](https://help.ubuntu.com/community/CronHowto)